x/spec/keeper/keeper.go

```diff
@@ -42,3 +42,5 @@ func NewKeeper(
 func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", fmt.Sprintf("x/%s", types.ModuleName))
 }
+
+func (k Keeper) BeginBlock(ctx sdk.Context) {}
```

Comment:
why is this needed? (the function does nothing)

Reply:
preparation for air-conditioning